src/genetic_forensic_portal/app/client/keycloak_client.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+from typing import Any
+
+from keycloak import KeycloakOpenID
+
+keycloak_openid = KeycloakOpenID(
+    server_url="http://localhost:8080/",
```

Comment:
```suggestion
    server_url=f"http://{os.environ.get('KEYCLOAK_HOST', 'localhost')}:{os.environ.get('KEYCLOAK_PORT', '8080')}/",
```
Maybe change this to be able to set from environment variable. Something like above, or maybe an option to directly grab the full server url from env variable. In case they end up liking keycloack and planning to use that. 

Or rather, does python keycloak already allow this functionality?

Reply:
Makes sense, can do!